template/macros.jinja

```diff
@@ -66,7 +71,7 @@
       {%- do fsl.append('') %}
     {%- endif %}
     {%- for fs in fsl %}
-      {%- for source_file in source_files %}
+      {%- for source_file in files %}
```

Comment:
For consistency: `source_file` => `file`.  What do you think?

Reply:
If doing that, then change below: `source_file.lstrip('/')` => `file.lstrip('/')`.

Reply:
Update: Forget these comments, I'd prefer to solve [the comment above](https://github.com/saltstack-formulas/template-formula/pull/77/files#r264497818) instead.

Reply:
Suggestion: `file` => `src_file`, in both places.

Reply:
Thanks.